fe/fe-core/src/main/java/com/starrocks/common/proc/FrontendsProcNode.java

```diff
@@ -60,7 +60,7 @@ public class FrontendsProcNode implements ProcNodeInterface {
 
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
             .add("Name").add("IP").add("EditLogPort").add("HttpPort").add("QueryPort").add("RpcPort")
-            .add("Role").add("ClusterId").add("Join").add("Alive").add("ReplayedJournalId")
```

Comment:
I think we can keep the ClusterId field.
1. to keep compatible
2. to know if two frontends are in the same cluster or not. Sometimes when a Frontend is started without helper, it will become leader. often we use clusterId to verify if they are in the same cluster or not.